### PR TITLE
[4.0] Article readmore in frontend

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #20477

When editing an article on the front end it ws not possible to insert a readmore link

This pr fixes that. Also test that it doesnt let you insert more than one readmore and that it still works in the admin

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 81fb6fb053d3e861ec47fcfe9c84001fc0449afa<br /><br />Only one Read-More is possible in Front- and Backend.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21207">issues.joomla.org/tracker/joomla-cms/21207</a>.</sub>

**Review by dgrammatiko — APPROVED** [review]

File: media/com_content/js/admin-article-readmore.es6.js [hunk 1]
```diff
@@ -4,24 +4,17 @@
...
   'use strict';
+  const options = window.Joomla.getOptions('xtd-readmore');
 
   window.insertReadmore = (editor) => {
-    if (!Joomla.getOptions('xtd-readmore')) {
-    // Something went wrong!
-      return false;
+    if (!options) {
+      // Something went wrong!
+      throw new Error('XTD Button read more not properly initialized');
```
Comment (Quy) [inline media/com_content/js/admin-article-readmore.es6.js]: Change to `\'read more\'` to be consistent with the other change.

**Comment by brianteeman** [comment]
drone error is unrelated to this pr

**Comment by TobsBobs** [comment]
screenshot_2018-07-27 home 1

Readmore work!

**Comment by TobsBobs** [comment]
I have tested this item :white_check_mark: successfully on e369757326bb1ab2fd2aa9ec3ae37457b220d038<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21207">issues.joomla.org/tracker/joomla-cms/21207</a>.</sub>

**Comment by TobsBobs** [comment]
More then one readmore is possible in frontend.

screenshot_2018-07-27 home 2

**Comment by TobsBobs** [comment]
> Also test that it doesnt let you insert more than one readmore and that it still works in the admin

unbenannt

Only one readmore is possible in front- and backend.

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 16f7a0b9711a08580f853bf3e1e4d03509d5f369<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21207">issues.joomla.org/tracker/joomla-cms/21207</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Comment by laoneo** [comment]
Thanks!

**Comment by Condorious** [comment]
Where we can get worked files?
Thanks!

**Comment by wojsmol** [comment]
Hi @Condorious 
Ether wait for next [nightly builds.](https://developer.joomla.org/nightly-builds.html) or [download 4.0-dev branch from github](https://github.com/joomla/joomla-cms/archive/4.0-dev.zip) and read [this](https://docs.joomla.org/J4.x:Setting_Up_Your_Local_Environment#Quick_start_guide)

**Comment by Condorious** [comment]
Thank you, @wojsmol

**Pull request merged**